UnityProject/Assets/Scripts/Core/Lighting/LightsHolder.cs

```diff
@@ -1,8 +1,10 @@
 ﻿using System;
+using System.Diagnostics;
 using System.Linq;
 using Light2D;
 using Mirror;
 using UnityEngine;
+using Debug = UnityEngine.Debug;
```

Comment:
What are you using these namespaces for?

Reply:
leftovers from testing. cleared them now